src/Entity/Release.php

```diff
@@ -95,7 +95,7 @@ class Release implements \JsonSerializable, \Stringable
     #[Assert\NotNull]
     #[ORM\ManyToOne(targetEntity: MajorVersion::class, inversedBy: 'releases')]
     #[ORM\JoinColumn(name: 'major_version', referencedColumnName: 'version')]
-    private MajorVersion $majorVersion;
+    private ?MajorVersion $majorVersion;
```

Comment:
this one needs to be set and cannot be null

Reply:
We are actually asserting it to be not null by [Assert\NotNull]

The database however can provide a null in which case the validation then would be triggered:

 Property App\Entity\Release::$majorVersion type mapping mismatch: database can contain App\Entity\MajorVersion|null but property expects App\Entity\MajorVersion.

Otherwise we would have to change the database mapping or accept a phpstan baseline

Reply:
I checked with @buchmarv, seems that we need to revalidate if all relations are defined properly. I think we can do this After the initial fix.